static/js/apps/explore/app.tsx

```diff
@@ -214,6 +216,13 @@ export function App(props: { isDemo: boolean }): JSX.Element {
       pageMetadata.pageConfig &&
       pageMetadata.pageConfig.categories
     ) {
+      // If the response is a single PLACE_OVERVIEW tile, redirect to the place explorer.
+      if (isPlaceOverviewOnly(pageMetadata)) {
+        console.log('Redirecting to place explorer');
+        let url = 'place/';
+        url += pageMetadata.pageConfig.metadata["placeDcid"][0];
```

Comment:
To avoid a possible array out of bounds, can you use `pageMetadata.place.dcid` instead of  `pageMetadata.pageConfig.metadata["placeDcid"][0]` ?  The former is the "primary place" detected in the query, which should always be the same as the place overview

Reply:
Prefer using const when possible. What do you think about:

```
const placeDcid = pageMetadata.place.dcid;
const url = `/place/${placeDcid}`;
```

Reply:
Done.